client/oci_env/commands.py

```diff
@@ -11,7 +11,15 @@ def exec(args, client):
 
 
 def db(args, client):
-    pass
+    if args.action == 'reset':
+        exit_if_failed(
+            client.exec_container_script(
+                f"database_reset.sh",
+                args=None,
+                interactive=True)
+        )
```

Comment:
adding `client.compose_command("restart")` will restart the services and you can get rid of the warning in the shell script

Reply:
done